src/ResourceManager/Insights/Commands.Insights/Diagnostics/GetAzureRmDiagnosticSettingCommand.cs

```diff
@@ -36,11 +35,21 @@ public class GetAzureRmDiagnosticSettingCommand : ManagementCmdletBase
         [ValidateNotNullOrEmpty]
         public string ResourceId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the diagnostics setting name parameter of the cmdlet
+        /// </summary>
+        [Parameter(Position = 1, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The diagnostic setting name. Defaults to 'service'")]
```

Comment:
@pkamath13 @gucalder optional parameters should not have a `Position` property

Reply:
Position removed.